src/status_im/communities/core.cljs

```diff
@@ -339,14 +339,19 @@
 
 (fx/defn edit-channel-pressed
   {:events [::edit-channel-pressed]}
-  [{:keys [db] :as cofx} community-id chat-name description color emoji]
-  (fx/merge cofx
-            {:db (assoc db :communities/create-channel {:name         chat-name
-                                                        :description  description
-                                                        :color        color
-                                                        :community-id community-id
-                                                        :emoji        emoji})}
-            (navigation/open-modal :edit-community-channel nil)))
+  [{:keys [db] :as cofx} community-id chat-name description color emoji chat-id]
+  (let [{:keys [color emoji]} (if (string/blank? emoji)
+                                (rand-nth emoji-thumbnail-styles/emoji-picker-default-thumbnails)
+                                {:color color :emoji emoji})]
+    (fx/merge cofx
+              {:db (-> db
+                       (assoc :current-chat-id chat-id)
```

Comment:
The current app is crashing while saving "edit channel values" because it is missing current-chat-id. I associated the chat-id.
(Do I also need to make an issue and link PR/Comment?)

Reply:
yes please

Reply:
and we need to fix it another way, because we shouldn't set :current-chat-id if we don't open chat

Reply:
How about as a field of "create-channel"? That is already being set for old values of channel name and colours.

Reply:
field as :edit-channel-id or :edit-chat-id